Remove expected test failures for cond

ydwu4 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #108709
* #107850
* #107662

Remove the expected failure in def test_control_flow_tracing(self) by chaning the error message to `Expected pred to be bool or tensor, but got Proxy\(eq\)`

ydwu4 committed:
Remove expected test failures for cond

[ghstack-poisoned]

ydwu4 committed:
Update on "Remove expected test failures for cond"


After #107850, we are able to remove the expected failures for make_fx(f, tracing_mode="symbolic") for f that contains cond. 

[ghstack-poisoned]

ydwu4 committed:
Update on "Remove expected test failures for cond"


After #107850, we are able to remove the expected failures for make_fx(f, tracing_mode="symbolic") for f that contains cond. 

[ghstack-poisoned]

ydwu4 committed:
Update on "Remove expected test failures for cond"


Remove the expected failure in def test_control_flow_tracing(self) by chaning the error message to `Expected pred to be bool or tensor, but got Proxy\(eq\)`

[ghstack-poisoned]

ydwu4 committed:
Update on "Remove expected test failures for cond"


Remove the expected failure in def test_control_flow_tracing(self) by chaning the error message to `Expected pred to be bool or tensor, but got Proxy\(eq\)`

[ghstack-poisoned]

ydwu4 commented:
@pytorchbot merge

huydhn commented:
@pytorchbot revert -m 'Sorry for reverting this, but test_export_with_symbool_inputs is failing in trunk https://hud.pytorch.org/pytorch/pytorch/commit/a08e1370ef8cb13cfbf18d9663427a57fa8657f2' -c landrace

ydwu4 committed:
Update on "Remove expected test failures for cond"


Remove the expected failure in def test_control_flow_tracing(self) by chaning the error message to `Expected pred to be bool or tensor, but got Proxy\(eq\)`

[ghstack-poisoned]

ydwu4 commented:
@pytorchbot merge
